### DISCUSSION TITLE
How to start the server?

### DISCUSSION THREAD

**Discussion by OleksandrMak** [inventree/InvenTree#4324] [General]
I made "quick-and-easy install" and it was successful on Debian 11 OS. I even connected to the server and made first tries. But after restart it don't work.  
I tried:
invoke worker 
received: "Can't find any collection named 'tasks'!".

I try to use it from /home/inventree$ 
ls -a command from this folder gives: .  ..  .bash_logout  .bashrc  .cache  .profile

**Comment by OleksandrMak** [comment]
I completely reinstall Inventree with OS. I installed in manually as BareMetal and I found my 1 half of answer to my question.

I need to run in terminal "sudo supervisord" to run supervisor for server. 
But I have another question. It doesn't startup automatically after restart. 
How to start supervisord at startup. As I see default config file it has such possibility (=true) but it doesn't work.

**Reply by matmair** [reply]
@OleksandrMak are you sure supervisord is starting up on starts? Check if it is registered as a service and enable `sudo service supervisord status` should show you if it is running and registered

**Reply by OleksandrMak** [reply]
I receive answer that this server not running.
>>>>>>Unit supervisord.service could not be found.

As I wrote above, I made compile reinstall and I used manual installation. 
If I will type in terminal 
sudo supervisord
It will run. But how to do it automatically when startup?

**Comment by matmair** [comment]
Hi there @OleksandrMak; Which instructions did you follow? Do you have a link? I am not certain what method you refer to by quick-and-easy.
If you used the install script/package installer: try running `inventree run worker` to start the worker process. In general the server should *just* start when rebooting - it could be that another webserver is already using the port on startup - that happens quite a lot.

**Reply by OleksandrMak** [reply]
Sorry for very late answer.
I used "Manual Bare Metal Install" [Manual Bare Metal Install](url https://docs.inventree.org/en/latest/start/intro/)

and also separete instruction for server: [](url https://docs.inventree.org/en/latest/start/production/)
